manifest.json

```diff
@@ -3,7 +3,7 @@
   "name": "Reddit to Teddit",
   "version": "0.0.1",
   "description": "Automatically redirect web pages from Reddit to Teddit.",
-   "permissions": ["tabs", "scripting"],
```

Comment:
Tabs requires more permissions, but since I'm using host permissions already to access the url this seems unnecessary, so I removed it.